src/content/core/user-roles-and-permissions.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: User Roles and Permissions in Okteto
+description: Understand User Roles and Permissions in the Okteto Platform
+sidebar_label: User Roles and Permissions
+id: user-roles-and-permissions
+---
+
+
+The Okteto platform offers a robust Role-Based Access Control (RBAC) system to manage and control access to different environments within the platform.
+This page outlines the roles, permissions, and key features available to Admins and Developers in Okteto.
+
+## User Roles
+
+In Okteto, there are two primary roles: **Admins** and **Developers**.
+
+## Admins
```

Comment:
This has the same level than `User Roles`, the title just before. Shouldn't we move things inside one level more? It feels odd when seeing in the browser

Reply:
Yes, thanks for spotting these